gcsio/src/test/java/com/google/cloud/hadoop/gcsio/integration/GoogleCloudStorageGrpcIntegrationTest.java

```diff
@@ -48,8 +49,11 @@ public class GoogleCloudStorageGrpcIntegrationTest {
 
   @Parameters
   // We want to test this entire class with both gRPC-LB and Traffic Director
+  // Some of our internal endpoints only work with TD
   public static Iterable<Boolean> tdEnabled() {
-    return Arrays.asList(false, true);
+    return Boolean.parseBoolean(System.getenv("GCS_TEST_ONLY_RUN_WITH_TD_ENABLED"))
```

Comment:
Looks like the intent is to run for `directpath` not TD

Can we modify the env var to `ENFORCE_DIRECT_PATH` ? and add a comment that enforcing direct path with internal endpoint is only supported on TD

Reply:
`ENFORCE_DIRECT_PATH` is one of the usecase we are facing which is independent of what we are testing here. This piece of code only specifies whether we want to test with tdEnabled or not.
I have added the comment in our google3 codebase where we use this enforce endpoint.

Reply:
What is the use-case to test TD alone?